src/store/builder/index.ts

```diff
@@ -140,11 +140,6 @@ export const mutations: MutationTree<IBuilderState> = {
     operations[operation.kind] = operation
   },
 
-  setBlockPositionTo(state, {position: {x, y}, block}) {
-    block.ui_metadata.canvas_coordinates.x = x
-    block.ui_metadata.canvas_coordinates.y = y
-  },
```

Comment:
Moved to `flow/flow_updateBlockCoordinates` to trigger a mutation that `undoRedo` can handle.

Reply:
Nice solution, I agree